src/helpers/bluez/postinstall.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+sudo ln -s /etc/sv/dbus /var/service/
+sudo ln -s  /etc/sv/bluetoothd /var/service/
+
+sudo usermod -a -G bluetooth $USER
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)